html-forms/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1><strong>Coding Form!</strong></h1>
+
+    <form name="survey">
+      <div id="emailField">
+        <label for="emailField">Email</label>
+      </div>
+
+      <input type="email" id="email" name="emailField" />
```

Comment:
The `id` of the `input` needs to match the `for` of the `label`. So this should be `for="email"` so it's linked to the `input` rather than the `div`